subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -319,22 +323,54 @@ int lll_deinit(void)
 
 int lll_csrand_get(void *buf, size_t len)
 {
+#if defined(CONFIG_ENTROPY_NRF5_RNG)
 	return entropy_get_entropy(dev_entropy, buf, len);
+#else
+	/* FIXME: No suitable entropy device available yet.
+	 *        It is required by Controller to use random numbers.
```

Comment:
Is this a requirement from the BT spec? 

If so, then uninitialized values are probably not a valid random value. 

When is this an issue? Do we not have entropy support for the nRF54L?

Reply:
I don't understand why it shouldn't return an error then. 

We are just returning uninitialized values and err == 0, which not only would be visible in e.g. valgrind, but also not complying with any potential randomness requirements. 

This does not look good to me, but I see that both @carlescufi and @aescolar have approved it, so maybe it's not a real issue.

Reply:
The scope of this PR is limited to preliminary support and stated in the commit as:

```
Bluetooth: Controller: Preliminary support for nRF54L15 SoC

Add preliminary support for nRF54L15 SoC. This commit does
not support Controller Random Number Generation and
Controller Cryptography (AES-128 encryption) commands, nor
does it support encrypted connections.
```

The board is also a Preview Dev Kit, hence, there will be updates to come in the future.

Reply:
Fair, had missed that. Thanks!